<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerManageClass</class>
 <widget class="QMainWindow" name="TimerManageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TimerManage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_chart">
      <property name="text">
       <string>统计数据</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_finish">
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_countTime">
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_list">
      <property name="text">
       <string>任务列表</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>Task</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../TimerManage.qrc"/>
 </resources>
 <connections/>
</ui>
